addon/services/preprint-word.js

```diff
@@ -0,0 +1,40 @@
+import Ember from 'ember';
+
+export default Ember.Service.extend({
+    i18n: Ember.inject.service(),
+    getPreprintWord() {
```

Comment:
This service doesn't do/provide anything that `i18n` doesn't already. What's the point of it?